src/In.ProjectEKA.HipService/Common/HttpRequestHelper.cs

```diff
@@ -13,7 +13,7 @@ public static class HttpRequestHelper
         public static HttpRequestMessage CreateHttpRequest<T>(
             string url,
             T content,
-            string token,
+            string token = null,
```

Comment:
Codacy found an issue: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5771073)